### DISCUSSION TITLE
Replacing IEDs with EODS IEDs

### DISCUSSION THREAD

**Discussion by SgtKrunch** [Vdauphin/HeartsAndMinds#1164] [Question and answer]
In the core/mission.sqf, line 386 for the IED classnames. If I switch those classnames to EODS IED classnames, will that cause any adverse effect on the placement or spawning of them? Would there be any negative consequences on the mission that would affect playability? I would try it out just to test, but I'm away from my computer for a while, so this is a hypothetical question.

**Comment by Vdauphin** [comment]
Hello,

> will that cause any adverse effect on the placement or spawning of them?

The spawning system is here:
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/ied/create.sqf#L49

 > Would there be any negative consequences on the mission that would affect playability?

I don't know EODS. There is nothing fancy. H&M only create mine. You should be allow to spawn any mine. The only issue could be the offset that you could tweak in mission parameters

Yes, try it in debug mode, you will see.

As reference: https://github.com/Vdauphin/HeartsAndMinds/issues/336

Cheers

**Reply by SgtKrunch** [reply]
> As reference: #336

I'll update you if I can get one of my guys to update our map with it and try it out.
